WalletWasabi.Fluent/ViewModels/AddWallet/WalletNamePageViewModel.cs

```diff
@@ -25,11 +25,17 @@ public WalletNamePageViewModel(WalletCreationOption creationOption, string? impo
 	{
 		_importFilePath = importFilePath;
 
+		if (creationOption == WalletCreationOption.AddNewWallet)
```

Comment:
Why is this check necessary? Shouldn't we pre-populate the wallet name for any `WalletCreationOption`?

Reply:
@yahiheb seems to me that trying to recover a wallet shouldn't give you `Wallet 2` as a default name. Though it probably applies for add HW wallet too

Reply:
> @yahiheb seems to me that trying to recover a wallet shouldn't give you `Wallet 2` as a default name.

Same can said about adding a new wallet. IMO we should pre-populate the wallet name for all or none.

Reply:
@ichthus1604, I agree with @yahiheb. We can revisit this in future.